React/my-app/src/components/Book/index.js

```diff
@@ -1,13 +1,16 @@
 import React from "react";
 import "./styles.css";
 import defaultImg from "../../assets/photos/default.png";
+import { Link } from "react-router-dom";
 
-const Book = ({ imageUrl, title, author }) => (
-  <div className="book_info">
-    <img className="cover" src={imageUrl ? imageUrl : defaultImg} />
-    <h1 className="title">{title}</h1>
-    <h1 className="author">{author}</h1>
-  </div>
+const Book = ({ id, imageUrl, title, author }) => (
+  <Link to={`/books/${id}`}>
+    <div className="book_info">
+      <img className="cover" src={imageUrl ? imageUrl : defaultImg} />
```

Comment:
instead of compare here, declare the next.
```
Book.defaultPropTypes = {
   imageUrl: defaultImg
}
```
So if imageUrl is null or undefined, component go to render default aautomatic

Reply:
I try to do this, but the wrapper dosen't send me an "undefined" wich activates the DefaultProps. It sends me a 'null'. Wich actually it's something so the DefaultProps dosen't activate. So instead i did the following:

in the wrapper i add a conditional.
```javascript
imageUrl={element.image_url || defaultImg}
```